Project2015To2017.Console/Options.cs

```diff
@@ -24,12 +24,16 @@ public class Options
 		[Option('o', "output-path", Default = false, HelpText = "Will not create a subfolder with the target framework in the output path")]
 		public bool NoTargetFrameworkToOutputPath { get; set; } = false;
 
+		[Option('z', "analyze-only", Default = false, HelpText = "Do only parsing and analysis steps, can be used even on already converted projects")]
+		public bool AnalyzeOnly { get; set; } = false;
```

Comment:
What's the difference between AnalyzerOnly and DryRun? They seem to be the same?

Reply:
DryRun only implies no writing will apply. AnalyzeOnly will also allow CPS project reading and analysis.
Analysis will also be ran as part of the conversion process, both before and after conversion (for future reference).

Reply:
doesn't that mean that we can collapse them into the same toggle? It doesn't make sense for dry run to just work for old projects?

Reply:
@hvanbakel uhm, suppose we might have post-conversion transformations. Like migration to .NET Core 3 (whatever) for users with CPS projects. That would complicate matters since we will want to have dry run for that scenario.